### PR TITLE
[W6.4h][W13-B1]Alicia Ho Sor Sian

### PR DESCRIPTION
**Pull request by novellius**
Implemented interface Printable for classes Name, Phone, Email, Address. Added methods getPrintableString() on the mentioned classes and in ReadOnlyPerson.java.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Interface that prints Person's personal Information, such as Address, Email, Name and Phone
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:4]: This interface need not be about person/addressbook at all!
References to addressbook or person can be removed here, just what it is supposed to do is enough!

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ */
+public interface Printable {
+    /**
+     * Produces a printable string representation of an object.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:8]: This is more appropriate! :+1:

**Comment by okkhoy** [comment]
@SuxianAlicia 
Some comments added. Please ack & close the PR after reading comments.

**Comment by novellius** [comment]
Noted. Thank you.

**Pull request closed without merging**